Source/messaging/MessageUnit.h

```diff
@@ -37,11 +37,17 @@ namespace WPEFramework {
         */
         class EXTERNAL MessageUnit : public Core::Messaging::IStore {
         public:
-            static constexpr uint16_t MetadataBufferSize = 8 * 1024;
+            static constexpr uint16_t MetadataBufferSize = 4 * 1024;
             static constexpr uint16_t TempMetadataBufferSize = 128;
             static constexpr uint16_t MaxDataBufferSize = 63 * 1024;
             static constexpr uint16_t TempDataBufferSize = 1024;
 
+            enum metadataFrameProtocol : uint8_t {
+                UPDATE      = 0,
```

Comment:
Like we discussed during our morning meeting, we no longer need this additional safety, since we have ASSERTs to check if the frame reader actually has data to read

Reply:
Correct.